src/accountsdb/db.zig

```diff
@@ -2364,7 +2369,7 @@ fn loadTestAccountsDB(allocator: std.mem.Allocator, use_disk: bool) !struct { Ac
     try accounts_db.loadFromSnapshot(
         snapshot.accounts_db_fields.file_map,
         accounts_dir,
-        1,
+        @intCast(n_threads),
```

Comment:
Pointless integer cast:
```suggestion
        n_threads,
```

Reply:
eb28c8e